kong/llm/drivers/shared.lua

```diff
@@ -553,6 +557,16 @@ function _M.post_request(conf, response_object)
   request_analytics_plugin[log_entry_keys.META_CONTAINER][log_entry_keys.REQUEST_MODEL] = kong.ctx.plugin.llm_model_requested or conf.model.name
   request_analytics_plugin[log_entry_keys.META_CONTAINER][log_entry_keys.RESPONSE_MODEL] = response_object.model or conf.model.name
 
+  -- Set the llm latency meta, and time per token usage
+  if kong.ctx.shared.ai_request_start_time then
+    local llm_latency = (ngx.now() - kong.ctx.shared.ai_request_start_time) * 1000
```

Comment:
let's also expose the substracted result in kong.ctx.shared, like kong.ctx.shared.ai_request_duration something

Reply:
Done